gammapy/time/plot.py

```diff
@@ -30,3 +28,140 @@ def plot_time_difference_distribution(time, ax=None):
 
     # TODO: implement!
     raise NotImplementedError
+
+def plot_light_curve(name_3FGL, time_start, time_end, ax = None):
+    """Plot flux as a function of time for a fermi 3FGL object.
+
+    Parameters
+    ----------
+    name_3FGL : `string`
+        The 3FGL catalog name of the object to plot
+    fermi_met_start : `~astropy.time.Time`
+        Astropy time object for the start of the light curve
```

Comment:
No need to write "Astropy time object" again, that's what the `~astropy.time.Time` in the previous line says.
Here I'd write:
```
start_time : `~astropy.time.Time`
    Light curve start time
```
i.e. also change the parameter name, because the user no longer has to give `MET`.